### DISCUSSION TITLE
页眉修改

### DISCUSSION THREAD

**Discussion by jqc25z** [sjtug/SJTUThesis#987] [提问] [answered]
![1700651021553](https://github.com/sjtug/SJTUThesis/assets/112072840/e30d9eec-c38f-4ae0-9000-d4f7011a47b8)
在修改完参考文献几个字的间距后，右上角页眉的参考文献也有了间距，怎么样可以保持页眉里的参考文献没有间距呀。还有页眉可以和word保持一致都是灰色吗

**Comment by LogCreative** [comment, marked as answer]
调整字间距的更改可能不是必要的。

<details>
<summary>
如果必须修改
</summary>

```latex

\clearpage
{

\ExplSyntaxOn
\bool_if:NTF \g__sjtu_twoside_bool
{
    \fancyhead [ LE ]     { 参考文献 }
    \fancyhead [ RO ]     { 参考文献 }
}
{
    \fancyhead [ R ] { 参考文献 }
}
\ExplSyntaxOff

% 参考文献
\printbibliography[heading=bibintoc]

\clearpage
}
```

</details>

Word 里页眉是灰色的说明你没有在编辑页眉，双击进去后你会发现页眉是黑色的，导出的 PDF 页眉也是黑色的。
